pcsx2/USB/USB.cpp

```diff
@@ -670,6 +670,34 @@ std::string USB::GetConfigSubKey(const std::string_view& device, const std::stri
 	return fmt::format("{}_{}", device, bind_name);
 }
 
+void USB::CopyConfiguration(SettingsInterface& dest_si, const SettingsInterface& src_si)
```

Comment:
You want to use the Copy routines here, not Set. Otherwise, it'll set values for keys which don't exist in the global configuration.

See https://github.com/PCSX2/pcsx2/blob/1b673d9dd0829a48f5f0b6604c1de2108e981399/pcsx2/PAD/Host/PAD.cpp#L510-L601

Reply:
Thanks for catching that! I updated it to use the copy routines instead.